cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -118,6 +118,11 @@ instance ( Consensus.ShelleyBasedEra era
                          , "delegationState" .= Shelley._delegationState lState
                          ]
 
+instance Crypto.Crypto crypto => ToJSON (ShelleyLedger.IncrementalStake crypto) where
+  toJSON iStake = object [ "stake" .= Map.toList (ShelleyLedger.credMap iStake)
```

Comment:
Nested `stake.stake` is a little confusing, maybe something like `stake.credentials` instead?

```suggestion
  toJSON iStake = object [ "credentails" .= Map.toList (ShelleyLedger.credMap iStake)
```

Reply:
I like this suggestion.